MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -147,15 +152,38 @@ object VisionManager {
         this.application = application
     }
 
-    /**
-     * Initialize SDK. Creates core services and allocates necessary resources.
-     * No-op if called while SDK is created already.
-     */
-    @JvmOverloads
-    @JvmStatic
-    fun create(
-        videoSource: VideoSource = Camera2VideoSourceImpl(application)
-    ) {
+    sealed class VisionMode {
+        data class Default(
+            private val application: Application,
```

Comment:
looks like `application` is overkill here

Reply:
It's needed for instantiating Camera2VideoSourceImpl, we can't use val of VisionManager since this class will be moved to the upper level.